### PR TITLE
woeusb: 3.3.1 -> 5.1.0

### PR DESCRIPTION
**Pull request by justinas**
###### Motivation for this change

WoeUSB has been [moved to its own organization](https://github.com/WoeUSB/WoeUSB) on GitHub and many new versions have been released since 3.3.1.

Note that `woeusbgui` is not being developed anymore and is not in the main repo. I am not sure if such a breaking change warrants something like a separate `woeusb3` derivation for those that do want the GUI.

There is a rewrite called [WoeUSB-ng](https://github.com/WoeUSB/WoeUSB-ng) which seems to provide both a CLI and a GUI. It probably should be packaged separately.

Creating and booting a Windows USB was successful:
```console
$ sudo ./woeusb -d Win10_20H2_v2_English_x64.iso /dev/sdX
```

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 123409` at b8a997a7 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>tests.trivial</li>
    <li>woeusb</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/tools/misc/woeusb/default.nix:25:3:
```
   |
25 |   installPhase = ''
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/stale-substitute.md">stale-substitute</a>

Stale substituteInPlace detected.
substituteStream(): WARNING: pattern '/media/' doesn't match anything in file '/nix/store/hxhmmdv6lxkbai6yi0rga4lvlaawhsgw-woeusb-5.1.0/bin/woeusb'
Near pkgs/tools/misc/woeusb/default.nix:47:0:
```
   |
47 |     description = "Create bootable USB disks from Windows ISO images";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/tools/misc/woeusb/default.nix [hunk 1]
```diff
@@ -53,7 +46,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Create bootable USB disks from Windows ISO images";
     homepage = "https://github.com/slacka/WoeUSB";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/woeusb/default.nix:48]:
```suggestion
    homepage = "https://github.com/WoeUSB/WoeUSB";
```

File: pkgs/tools/misc/woeusb/default.nix [hunk 2]
```diff
@@ -1,45 +1,38 @@
...
-  postInstall = ''
-    # don't write data into /
-    substituteInPlace "$out/bin/woeusb" \
-      --replace /media/ /run/woeusb/
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/share/man/man1
+    mv sbin/woeusb $out/bin
+    mv share/man/man1/woeusb.1 $out/share/man/man1
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/woeusb/default.nix:30]: Please use installManPage.

File: pkgs/tools/misc/woeusb/default.nix [hunk 3]
```diff
@@ -1,45 +1,38 @@
...
   '';
 
-  postInstall = ''
-    # don't write data into /
-    substituteInPlace "$out/bin/woeusb" \
-      --replace /media/ /run/woeusb/
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/share/man/man1
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/woeusb/default.nix:28]:
```suggestion
```

**Pull request merged**